_sass/_home.scss

```diff
@@ -0,0 +1,128 @@
+@import "bootstrap/buttons";
+@import "bootstrap/code";
+@import "bootstrap/images";
+@import "masthead";
+@import "featurettes";
+@import "syntax";
+@import "highlight";
+
+.social {
+  list-style: none;
+  margin: 1.25rem 0 0 0;
+  padding: 0;
+  text-align: center;
+
+  li {
+    display: inline;
+
+    &:not(:last-child) {
+      margin-right: 24px;
+    }
+
+    svg {
+      width: 24px;
+      height: 24px;
+
+      path,
+      circle {
+        transition: fill .25s ease-in-out;
+        fill: $body-color;
+      }
+    }
+
+    a:focus,
+    a:hover {
+      cursor: pointer;
+      text-decoration: none;
+
+      svg {
+        width: 24px;
+        height: 24px;
+
+        path,
+        circle {
+          fill: $brand-primary;
+        }
+      }
+    }
+  }
+}
+
+.features {
+  list-style: none;
+
+  li {
+    padding-left: 1ch;
+    text-indent: -3ch;
+  }
+  li::before {
+    padding-right: 1ch;
+    content: "❖";
+  }
+}
+
+.home {
+  .content {
+    padding: 1.5rem 0 1.5rem;
+  }
+
+  span {
+    // display: block;
+  }
+
+  .lead {
+    font-size: 1.25rem;
+    font-weight: 300;
+  }
+
+  .welcome {
+    text-align: justify;
+  }
+
```

Comment:
```suggestion
```

> It makes sense, then, that the pineapple worked its way into art and architecture. Beyond wealth and luxury and novelty it also came to represent hospitality.<br><br>And so they were used, more often than not, to decorate entrances.
> &mdash; The Cultural Tutor (@culturaltutor) <a href="https://twitter.com/culturaltutor/status/1715530237289619949?ref_src=twsrc%5Etfw">October 21, 2023</a>

Reply:
<blockquote class="twitter-tweet"><p lang="en" dir="ltr">But even when the pineapple itself ceased to be an item of quite such luxury it endured as a symbol and became a standard decorative feature.<br><br>Hence something like the Duke of Westminster&#39;s house in France, from 1911. How to decorate your gateway? With pineapples, of course. <a href="https://t.co/yiSuvcF11s">pic.twitter.com/yiSuvcF11s</a></p>&mdash; The Cultural Tutor (@culturaltutor) <a href="https://twitter.com/culturaltutor/status/1715530239432917029?ref_src=twsrc%5Etfw">October 21, 2023</a></blockquote>

Reply:
> Thus the pineapple entered the design language of Baroque and Neoclassical architecture; it had become one potential decorative motif among many.<br><br>The entrance to the Examination Schools in Oxford, built in the 1870s, is another example of its use in gateways.
>  
> <a href="https://t.co/P0SNMcpglz">![F868XZlXEAAcIeg](https://github.com/OpenINF/openinf.github.io/assets/17770407/895825bc-5532-4417-9acb-48c2932c4e52)</a></p>&mdash; The Cultural Tutor (@culturaltutor) <a href="https://twitter.com/culturaltutor/status/1715530241848787414?ref_src=twsrc%5Etfw">October 21, 2023</a>